app/components/Onboarding/Steps/ConnectionDetails.js

```diff
@@ -1,10 +1,21 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import { FormattedMessage } from 'react-intl'
-import { Box } from 'rebass'
-import { Bar, Form, Header, Input } from 'components/UI'
+import { Box, Flex } from 'rebass'
+import { Bar, Form, Header, Input, Button } from 'components/UI'
+import WithSelectDir from 'components/withSelectDir'
 import messages from './messages'
 
+const SelectFileButton = ({ onClick }) => (
```

Comment:
Lets extract and move this component to `components/UI/SelectFileButton.js`

Reply:
Makes sense